homework/object-oriented-programming/src/main/java/com/cbfacademy/shapes/Shapes.java

```diff
@@ -0,0 +1,31 @@
+package com.cbfacademy.shapes;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Shapes {
+
+    public void computeArea(List<Enclosure> enclosure){
+       //ArrayList<Double> shapesArea = new ArrayList<>();
+        //take a list of object that implement enclosure - as a parameter
+        // use area method on this group of objects then add the area for each shape together   
+        for (int i = 0; i < enclosure.size(); i++) {
+           //
+        }
```

Comment:
You've described the steps correctly in your comment, so you just have to implement them in your code.

1. How do you get each enclosure from your loop? Could it be simplified by using the enhanced loop syntax?
2. Once you have a reference to the current `Enclosure` instance, how do you add it's area to a running total?